app/views/omni_nav/_search_box.html.erb

```diff
@@ -1,5 +1,5 @@
 <form class="omni-nav-search-box">
-  <input type="text" placeholder="Search" class="omni-nav-search-input">
+  <input type="text" placeholder="Search" id="search-input" class="omni-nav-search-input">
   <span class="search-icon">
     <svg viewBox="0 0 16 16"><path d="M16 13.5l-4.695-4.695c0.444-0.837 0.695-1.792 0.695-2.805 0-3.314-2.686-6-6-6s-6 2.686-6 6 2.686 6 6 6c1.013 0 1.968-0.252 2.805-0.695l4.695 4.695 2.5-2.5zM2 6c0-2.209 1.791-4 4-4s4 1.791 4 4-1.791 4-4 4-4-1.791-4-4z"></path></svg>
   </span>
```

Comment:
I think a partial at only 6 lines is overkill here. Any objections to moving this into the parent?